### PR TITLE
Add quirk for Schneider Electric NHROTARY/DIMMER/1

### PR DESCRIPTION
**Pull request by dflemstr**
## Proposed change

Adds support for Schneider Electric NHROTARY/DIMMER/1.  The main missing piece of functionality is documented in issue #2798.

## Additional information

Builds on #2928, I will rebase this PR once that PR is merged.  I didn't want to blindly cherry pick and squash changes from the other PR so that merging them remains easy.

Asking for help from @julienhd to help validate this PR.

## Checklist

- [x] The changes are tested and work correctly
- [x] `pre-commit` checks pass / the code has been formatted using Black
- [x] Tests have been added to verify that the new code works

### CODE REVIEW

**Comment by TheJulianJES** [comment]
- https://github.com/zigpy/zha-device-handlers/pull/2928 is merged now

**Comment by dflemstr** [comment]
Rebased onto upstream `dev` now

File: zhaquirks/schneiderelectric/dimmers.py
Comment (TheJulianJES) [inline zhaquirks/schneiderelectric/dimmers.py]: Use this here:
```python
PROFILE_ID: zgp.PROFILE_ID,
DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
```

File: zhaquirks/schneiderelectric/dimmers.py
Comment (TheJulianJES) [inline zhaquirks/schneiderelectric/dimmers.py]: Use this here:
```python
PROFILE_ID: zgp.PROFILE_ID,
DEVICE_TYPE: zgp.DeviceType.PROXY_BASIC,
```

File: zhaquirks/schneiderelectric/dimmers.py
Comment (dflemstr) [reply zhaquirks/schneiderelectric/dimmers.py]: Thanks, fixed

File: zhaquirks/schneiderelectric/dimmers.py
Comment (dflemstr) [reply zhaquirks/schneiderelectric/dimmers.py]: Thanks, fixed

**Review by TheJulianJES — APPROVED** [review]
Thanks, LGTM!
Should we wait for testing/approval from anyone else?

**Comment by dflemstr** [comment]
I think it is good to merge, since this change only exposes new attributes and doesn't change any existing behavior. I would like to test the integration with other HA components on top of this new attribute, and it would be nice to get this merged so I don't have to maintain my own local HA quirk

**Comment by TheJulianJES** [comment]
The quirk will likely only land in Home Assistant Core 2024.3.0 (betas and newer versions) due to the current release schedule/procedure.

**Comment by TheJulianJES** [comment]
The wiki can probably be also updated: https://github.com/zigpy/zha-device-handlers/wiki/Schneider-Electric

**Pull request merged**
